AutoCollection/RequestParser.ts

```diff
@@ -30,7 +30,7 @@ abstract class RequestParser {
             } else if (error instanceof Error) {
                 properties["error"] = error.message;
             } else if (typeof error === "object") {
-                for (var key in error) {
+                for (var key in <any>error) {
```

Comment:
it causes a build error for me. Probably different typescript version. I've removed it